srcpkgs/clion/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'clion'
+_dlname="CLion"
+_pkgname="clion"
+pkgname=clion
+version=2019.1.2
+revision=1
+archs="x86_64"
+depends="virtual?java-environment giflib libXtst"
+short_desc="CLion: A smart cross-platform IDE for C and C++"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="Commercial"
+homepage="https://www.jetbrains.org/"
+distfiles="https://download.jetbrains.com/cpp/${_dlname}-${version}.tar.gz"
+checksum=e8efbf3ec4aed7ef1e0eb8139e799fb2975e4c6363f2ac7504864f96d5aa8e74
+repository=nonfree
+nopie=yes
+
+post_extract() {
+	rm -rf _tmp
+	mkdir _tmp
+	# JetBrains' JRE can be installed separately as a subpackage.
+	# For now, just move it out of the way
+	mv jre64 _tmp/
+
+	# JetBrains' LLDB, GDB, and CMake can be made installable via separate subpackages
+	mv bin/lldb bin/gdb bin/cmake _tmp/
+}
+
+do_install() {
+	vmkdir usr/bin
+	vmkdir usr/share/doc/${_pkgname}
+	vmkdir usr/lib/${_pkgname}
+
+	mv bin lib plugins product-info.json build.txt ${DESTDIR}/usr/lib/${_pkgname}
+	mv license ${DESTDIR}/usr/share/doc/${_pkgname}
+
+	rm ${DESTDIR}/usr/lib/${_pkgname}/bin/fsnotifier-arm -rf
+	rm ${DESTDIR}/usr/lib/${_pkgname}/lib/pty4j-native/linux/ppc64le -rf
+
+	# Remove files for other CPU architectures
+	case $XBPS_TARGET_MACHINE in
+		x86_64)
+		rm ${DESTDIR}/usr/lib/${_pkgname}/bin/fsnotifier
+		rm ${DESTDIR}/usr/lib/${_pkgname}/bin/${_pkgname}.vmoptions
+		rm ${DESTDIR}/usr/lib/${_pkgname}/bin/libyjpagent-linux.so
+		rm ${DESTDIR}/usr/lib/${_pkgname}/lib/pty4j-native/linux/x86 -rf
+		;;
+		i686)
```

Comment:
That won't happen because of `archs=` set before.

Reply:
Amended the archs list.